code/src/components/HappyThought.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import moment from 'moment';
+import './HappyThought.css';
+
+export const HappyThought = ({
+  id,
+  message,
+  hearts,
+  createdAt,
+  onLikedThought,
+}) => {
+  const likeThoughtURL = `https://happy-thoughts-technigo.herokuapp.com/thoughts/${id}/like`;
+
+  const handleClick = (event) => {
+    event.preventDefault(); // To prevent refresh of window at click of button
+
+    fetch(likeThoughtURL, {
+      method: 'POST',
+      headers: {
+        'Content-type': 'application/json',
+      },
+    }).then(() => {
+      onLikedThought(id);
+    });
+  };
+
+  return (
+    <>
```

Comment:
No need for React Fragment (`</>`) if you return only one element (is your case `article`)